tests/automount/test_automountkey.yml

```diff
@@ -0,0 +1,180 @@
+---
+- name: Test automountmap
+  hosts: ipaserver
+  become: true
+  gather_facts: false
+
+  tasks:
+  - name: ensure location TestLocation is absent
+    ipaautomountlocation:
+      ipaadmin_password: SomeADMINpassword
+      name: TestLocation
+      state: absent
+
+#  - name: ensure map TestMap is absent
+ #   ipaautomountmap:
+ #     ipaadmin_password: SomeADMINpassword
+ #     name: TestMap
+ #     location: TestLocation
+ #     state: absent
+#
+#  - name: ensure key TestKey is absent
+#    ipaautomountkey:
+#      ipaadmin_password: SomeADMINpassword
+#      automountlocationcn: TestLocation
+#      automountmapname: TestMap
+#      automountkey: TestKey
+#      state: absent
```

Comment:
why are this tasks commented out? I think they should be used to ensure a clean test environment.